Loop/Views/SimpleBolusView.swift

```diff
@@ -0,0 +1,349 @@
+            )
+        case .carbEntrySizeTooLarge:
+            let message = String(
+                format: NSLocalizedString("The maximum allowed amount is %1$@ grams", comment: "Alert body displayed for quantity greater than max (1: maximum quantity in grams)"),
+                NumberFormatter.localizedString(from: NSNumber(value: LoopConstants.maxCarbEntryQuantity.doubleValue(for: .gram())), number: .none)
+            )
+            return SwiftUI.Alert(
+                title: Text("Carb Entry Too Large", comment: "Alert title for a carb entry too large error"),
+                message: Text(message)
+            )
+        case .manualGlucoseEntryOutOfAcceptableRange:
+            let formatter = QuantityFormatter(for: viewModel.glucoseUnit)
+            let acceptableLowerBound = formatter.string(from: LoopConstants.validManualGlucoseEntryRange.lowerBound, for: viewModel.glucoseUnit) ?? String(describing: LoopConstants.validManualGlucoseEntryRange.lowerBound)
+            let acceptableUpperBound = formatter.string(from: LoopConstants.validManualGlucoseEntryRange.upperBound, for: viewModel.glucoseUnit) ?? String(describing: LoopConstants.validManualGlucoseEntryRange.upperBound)
+            return SwiftUI.Alert(
+                title: Text("Glucose Entry Out of Range", comment: "Alert title for a manual glucose entry out of range error"),
+                message: Text(String(format: NSLocalizedString("A manual glucose entry must be between %1$@ and %1$@", comment: "Alert message for a manual glucose entry out of range error. (1: acceptable lower bound) (2: acceptable upper bound)"), acceptableLowerBound, acceptableUpperBound))
+            )
+        case .manualGlucoseEntryPersistenceFailure:
+            return SwiftUI.Alert(
+                title: Text("Unable to Save Manual Glucose Entry", comment: "Alert title for a manual glucose entry persistence error"),
+                message: Text("An error occurred while trying to save your manual glucose entry.", comment: "Alert message for a manual glucose entry persistence error")
+            )
+        case .infoPopup:
+            return closedLoopOffInformationalModal()
+        }
+        
+    }
+    
+    private func warning(for notice: SimpleBolusViewModel.Notice) -> some View {
+        switch notice {
+        case .glucoseBelowSuspendThreshold:
+            let suspendThresholdString = QuantityFormatter().string(from: viewModel.suspendThreshold, for: viewModel.glucoseUnit) ?? String(describing: viewModel.suspendThreshold)
+            return WarningView(
+                title: Text("No Bolus Recommended", comment: "Title for bolus screen notice when no bolus is recommended"),
+                caption: Text("Your glucose is below your suspend threshold, \(suspendThresholdString).", comment: "Caption for bolus screen notice when no bolus is recommended due input value below suspend threshold")
```

Comment:
Consider changing this to using a string format instead of string interpolation (for localization considerations)